pkg/util/envutil/env.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2023 The envd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package envutil
+
+import (
+	"os"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+)
+
+func GetDurationWithDefault(key string, o time.Duration) time.Duration {
+	v, found := os.LookupEnv(key)
+	if found && v != "" {
+		d, err := time.ParseDuration(v)
+		if err == nil {
+			return d
+		}
+		log.WithField(key, v).WithError(err).Panic("failed to parse")
```

Comment:
will `Panic()` panic here?

Reply:
Yes it's intentional since we don't want users to assume that the incorrect config is applied when actually not.